src/Nri/Ui/Data/PremiumLevel.elm

```diff
@@ -44,9 +45,12 @@ order : PremiumLevel -> Int
 order privileges =
     case privileges of
         PremiumWithWriting ->
-            2
+            3
 
         Premium ->
+            2
+
+        PremiumPromo ->
             1
```

Comment:
I think it's better to use `3` for PremiumPromo and leave the rest of the values as-is, since that's [what we're going for in the backend](https://linear.app/noredink/issue/GRO-173#comment-f559d70e)

```suggestion
        PremiumPromo ->
            3
            
        PremiumWithWriting ->
            2
            
        Premium ->
            1
```